### PR TITLE
UCS/SYS: Set names for threads

### PR DESCRIPTION
**Pull request by yosefe**
## Why
To allow quick identification of the various service threads in gdb

### CODE REVIEW

**Review by hoopoepg — APPROVED** [review]

**Comment by hoopoepg** [comment]
bot:pipe:retest

File: src/ucs/sys/sys.h [hunk 1]
```diff
@@ -626,6 +626,21 @@ ucs_status_t ucs_sys_get_file_time(const char *name, ucs_sys_file_time_t type,
...
+
+/*
+ * Create a named thread.
+ *
+ * @param [out] thread_id_p     If successful, set to the new thread id.
+ * @param [in]  start_routine   Thread function.
+ * @param [in]  arg             Custom argument for start_routine.
+ * @param [in]  fmt             Thread name format setring.
+ *
+ * @return UCS_OK if successful, or error status is failed.
```
Comment (brminich) [inline src/ucs/sys/sys.h:638]: `if failed`?

File: src/ucs/sys/sys.h [hunk 2]
```diff
@@ -626,6 +626,21 @@ ucs_status_t ucs_sys_get_file_time(const char *name, ucs_sys_file_time_t type,
...
 ucs_status_t ucs_sys_check_fd_limit_per_process();
 
+
+/*
+ * Create a named thread.
+ *
+ * @param [out] thread_id_p     If successful, set to the new thread id.
+ * @param [in]  start_routine   Thread function.
+ * @param [in]  arg             Custom argument for start_routine.
+ * @param [in]  fmt             Thread name format setring.
```
Comment (lyu) [inline src/ucs/sys/sys.h:636]: setring -> string?

**Comment by yosefe** [comment]
@brminich @lyu fixed

**Review by brminich — APPROVED** [review]

**Pull request merged**